roles/kubernetes-apps/ansible/tasks/main.yml

```diff
@@ -11,68 +11,16 @@
   delay: 2
   when: inventory_hostname == groups['kube-master'][0]
 
-- name: Kubernetes Apps | Delete old kubedns resources
-  kube:
-    name: "kubedns"
-    namespace: "{{ system_namespace }}"
-    kubectl: "{{ bin_dir }}/kubectl"
-    resource: "{{ item }}"
-    state: absent
-  with_items:
-    - 'deploy'
-    - 'svc'
-  tags:
-    - upgrade
-
-- name: Kubernetes Apps | Delete kubeadm kubedns
-  kube:
-    name: "kubedns"
-    namespace: "{{ system_namespace }}"
-    kubectl: "{{ bin_dir }}/kubectl"
-    resource: "deploy"
-    state: absent
+- name: Kubernetes Apps | DNS
+  include: tasks/{{ dns_mode }}.yml
```

Comment:
it shoud be include_tasks: "tasks/{{ dns_mode }}.yml"

Reply:
Fixed

Reply:
Reverted this change. It makes the cleanup jobs not to run.